src/test/fuzz.zig

```diff
@@ -0,0 +1,47 @@
+//! Utils functions for writing fuzzers.
+
+const std = @import("std");
+const assert = std.debug.assert;
+
+/// Returns an integer of type `T` with an exponential distribution of rate `avg`.
+/// Note: If you specify a very high rate then `std.math.maxInt(T)` may be over-represented.
+pub fn random_int_exp(random: std.rand.Random, comptime T: type, avg: T) T {
```

Comment:
Let's be more precise: `random_int_exponential`.

Reply:
Ok :)

I was copying Random.floatExp.

Reply:
Ah, fair enough. We typically spell out full words when it isn't too painfully long, but since it is mirroring a stdlib API that's fine either way. Come to think of it we do have others like this, e.g. `div_ceil`.